pkg/scheduler/api/shared_device_pool.go

```diff
@@ -0,0 +1,25 @@
+package api
```

Comment:
please add the copyright and change the name of this file to device_interface.go